crypto/x509/x_name.c

```diff
@@ -479,14 +479,15 @@ static int i2d_name_canon(STACK_OF(STACK_OF_X509_NAME_ENTRY) * _intname,
     return len;
 }
 
-int X509_NAME_set(X509_NAME **xn, X509_NAME *name)
+int X509_NAME_set(X509_NAME **xn, const X509_NAME *name)
 {
+    X509_NAME *name_copy;
```

Comment:
while you are modifying  this line, please add a blank after declaration

Reply:
No problem, done here and for further such occurrences in the same file.